src/IncludePanel.php

```diff
@@ -0,0 +1,201 @@
+<?php
+/**
+ * CakePHP(tm) : Rapid Development Framework (http://cakephp.org)
+ * Copyright (c) Cake Software Foundation, Inc. (http://cakefoundation.org)
+ *
+ * Licensed under The MIT License
+ * Redistributions of files must retain the above copyright notice.
+ *
+ * @copyright     Copyright (c) Cake Software Foundation, Inc. (http://cakefoundation.org)
+ * @link          http://cakephp.org CakePHP(tm) Project
+ * @license       http://www.opensource.org/licenses/mit-license.php MIT License
+ */
+namespace DebugKit;
+
+use Cake\Core\Plugin;
+use Cake\Event\Event;
+use Cake\Utility\Hash;
+use Composer\Json\JsonFile;
+use DebugKit\DebugPanel;
+
+/**
+ * Provides a list of files for the current request
+ *
+ */
+class IncludePanel extends DebugPanel
```

Comment:
Can this problem be solved with composition instead of inheritance? Having two classes named IncludePanel is awkward.

Reply:
done.